rpcserverhelp.go

```diff
@@ -236,6 +236,25 @@ var helpDescsEnUS = map[string]string{
 	"getblock--condition1": "verbose=true",
 	"getblock--result0":    "Hex-encoded bytes of the serialized block",
 
+	// GetBlockchainInfoCmd help.
+	"getblockchaininfo--synopsis": "Returns information about the current state of the block chain.",
+
+	// GetBlockchainInfoResult help.
+	"getblockchaininforesult-chain":                "The current network name.",
+	"getblockchaininforesult-blocks":               "The block count.",
+	"getblockchaininforesult-headers":              "The block header count.",
```

Comment:
Perhaps:

```The number of validated block headers that comprise the target best chain.```

I'm making this distinction because, while it's not currently the case, the plan is to move to a full headers-first download mode where all of the headers will be downloaded first followed by downloading all of the actual block data in parallel.